validator/src/bin/solana-test-validator.rs

```diff
@@ -511,6 +513,17 @@ fn main() {
         }
     }
 
+    if clone_feature_set {
+        if let Err(e) = genesis.clone_feature_set(
+            cluster_rpc_client
+                .as_ref()
+                .expect("bug: --url argument missing?"),
```

Comment:
This message doesn't seem correct, since the `clone_feature_set` arg requires `json_rpc_url`.
I know this is consistent with the other cloning args... perhaps we could clean up everywhere?

Reply:
I thought this was a bit strange too. Just so I'm sure I understand, are you suggesting moving to `unwrap()`s everywhere instead?

Reply:
> Just so I'm sure I understand, are you suggesting moving to unwrap()s everywhere instead?

Or `expect()`s with a truthful message. Perhaps something like "`json_rpc_url` input is required, so RpcClient construction should not fail", though that seems kind of wordy.